src/Cache/CacheClient.php

```diff
@@ -1841,6 +1842,57 @@ public function sortedSetFetchByRank(string $cacheName, string $sortedSetName, ?
         return $this->sortedSetFetchByRankAsync($cacheName, $sortedSetName, $startRank, $endRank, $order)->wait();
     }
 
+    /**
+     * Get the score associated with a value in a sorted set.
+     *
+     * @param string $cacheName Name of the cache that contains the sorted set.
+     * @param string $sortedSetName The sorted set to fetch the score from.
+     * @param string $value The value to fetch the score for.
+     * @return ResponseFuture<SortedSetGetScoreResponse> A waitable future which will provide the result of the sorted set get score operation upon a blocking call to wait:<br />
+     * <code>$response = $responseFuture->wait();</code><br />
+     * The response represents the result of the sorted set get score operation. This result is resolved to a type-safe object of one of the following types:<br>
+     * * SortedSetGetScoreHit<br>
+     * * SortedSetGetScoreMiss<br>
+     * * SortedSetGetScoreError<br>
+     * Pattern matching can be to operate on the appropriate subtype:<br>
+     * <code>
+     *     if ($hit = $response->asHit()) {
```

Comment:
Can we add the handling "miss" response type case here?

Reply:
Done in [123cf56](https://github.com/momentohq/client-sdk-php/pull/221/commits/123cf56528e635c20030822498747ea0e1ee0729)